src/m365/planner/commands/roster/roster-remove.ts

```diff
@@ -0,0 +1,90 @@
+import { Cli } from '../../../../cli/Cli';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request, { CliRequestOptions } from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+  confirm?: boolean;
+}
+
+class PlannerRosterRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ROSTER_REMOVE;
+  }
+
+  public get description(): string {
+    return 'Removes a Microsoft Planner Roster';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        confirm: !!args.options.confirm
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --id <id>'
```

Comment:
Either we use this short everywhere or nowhere. Since it's not in the other specs right now I'd say let's leave it for now, we can always add it later on.